py3status/modules/uptime.py

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8 -*-
+
+"""
+Display the system's uptime
+
+Configuration parameters:
+        - format : what will be displayed, formatted with predefined
+          placeholders; defaults to "{days} days, {hours} hours, {minutes} min"
+
+Available placeholders in format string:
+        - {days}: days part of the uptime
+        - {hours}: hours part of the uptime
+        - {minutes}: minutes part of the uptime
+        - {seconds}: seconds part of the uptime
+        - No {year} provided !
```

Comment:
ha ha

I think you should either add a year or remove the **this is not an option**

good hygiene suggests a year uptime should be avoided